cluster/addons/calico-policy-controller/calico-node-vertical-autoscaler-deployment.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     addonmanager.kubernetes.io/mode: Reconcile
 spec:
   replicas: 1
+  selector:
```

Comment:
Just curious, is this change fixing an actual issue or just making the manifests more readable?